### ISSUE TITLE
/boot/config.txt moved

### ISSUE THREAD

**Issue by N1-TR0** [pimoroni/pirate-audio#101]
/boot/config.txt has been moved to /boot/firmware/config.txt in the recent Raspberry Pi OS Update

**Comment by VaporwareII** [comment]
yeah I had to edit the install script but it worked fine after that.

**Comment by N1-TR0** [comment]
I was jus trying to make the devs apply this quick fix
